.github/workflows/balancer-js.yaml

```diff
@@ -79,7 +79,9 @@ jobs:
       - name: Compile
         run: yarn build
       - name: Run node in background for integration tests
-        run: npx hardhat node --hostname 127.0.0.1 --fork ${{ secrets.ALCHEMY_URL }} &
+        run: npx hardhat node TS_NODE_PROJECT='tsconfig.testing.json' --hostname 127.0.0.1 --fork ${{ secrets.ALCHEMY_URL }}
+      - name: Run goerli node in background for integration tests
+        run: npx hardhat node TS_NODE_PROJECT='tsconfig.testing.json' --hostname 127.0.0.1 --fork ${{ secrets.ALCHEMY_URL_GOERLI }} --port 8000 --config hardhat.config.goerli.ts
```

Comment:
I think we had issues accessing secrets in the past and we need to add them in the `env:` section. See [this](https://docs.github.com/en/actions/security-guides/encrypted-secrets#accessing-your-secrets). So would need to add `ALCHEMY_URL_GOERLI`.